src/overlays/actors/ovl_Boss_Sst/z_boss_sst.h

```diff
@@ -6,11 +6,63 @@
 
 struct BossSst;
 
+typedef void (*BossSstActionFunc)(struct BossSst*, GlobalContext*);
+
+#define BONGO_BOSSROOM_CENTER_X -50.0f
+#define BONGO_BOSSROOM_CENTER_Y 0.0f
+#define BONGO_BOSSROOM_CENTER_Z 0.0f
+#define BONGO_DRUM_RADIUS 550.0f;
```

Comment:
do these get used by the drum? if not seems like something that should be internal